online/main.yaml

```
@@ -101,6 +101,131 @@ sources:
         description: zoom resolution of the dataset
         type: int
     # metadata: (see EU catalog)
+
+  icon_d3hp003feb:
+    args:
+      chunks: null
+      consolidated: true
+      urlpath: https://s3.eu-dkrz-1.dkrz.cloud/wrcp-hackathon/data/ICON/d3hp003feb.zarr/{{ time_method }}_z{{ zoom }}_atm
```

is there something missing?

```suggestion
      urlpath: https://s3.eu-dkrz-1.dkrz.cloud/wrcp-hackathon/data/ICON/d3hp003feb.zarr/{{ time }}_{{ time_method }}_z{{ zoom }}_atm
```

---

2x  Thanks! 🙈